src/main/java/com/google/protobuf/maven/ProtocTestCompileNanoMojo.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2015 CNH Industrial NV. All rights reserved.
+ *
+ * This software contains proprietary information of CNH Industrial NV. Neither
+ * receipt nor possession thereof confers any right to reproduce, use, or
+ * disclose in whole or in part any such information without written
+ * authorization from CNH Industrial NV.
+ */
+
+package com.google.protobuf.maven;
+
+import java.io.File;
+
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugins.annotations.LifecyclePhase;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.plugins.annotations.ResolutionScope;
+
+/**
+ * This mojo executes the {@code protoc} compiler for generating test Java sources
+ * from protocol buffer definitions. It also searches dependency artifacts in the test scope for
+ * {@code .proto} files and includes them in the {@code proto_path} so that they can be
+ * referenced. Finally, it adds the {@code .proto} files to the project as test resources so
+ * that they can be included in the test-jar artifact.
+ *
+ * @since 0.3.3
```

Comment:
the next version will be 0.4.3
This goes into generated plugin documentation, and having correct version in ```@since``` is important